audio-analysis/main.py

```diff
@@ -11,18 +11,22 @@ def saveVideo(url, tag):
     video.getTextAnalysis()
     video.getAmplitudeAnalysis()
 
+    #For testing, will be removed after PR is approved
```

Comment:
Add an `if (TESTING) {` so you don't need to delete it

Reply:
True, was just trying to keep it consistent with how @mattlianje did it. I'll add the constant though since it makes more sense